deploy/library/postgresql_exec.py

```diff
@@ -135,7 +135,7 @@ def main():
         "port":"port",
         "db":"database"
     }
-    kw = dict( (params_map[k], v) for (k, v) in module.params.iteritems()
+    kw = dict( (params_map[k], v) for (k, v) in module.params.items()
```

Comment:
Was this to support Python 3?  I believe it is usually actually run on Python 2.x in our environment, and in that environment iteritems is more efficient, but if we need to change it to support 3, that's fine too.

Reply:
Yes; at some point in the testing process, `ansible-playbook` or `jinja2` was running under Python 3 and complained about the odbc templates.  Unless the collections are very, very long, I doubt we'll even notice the efficiency difference.